packages/nimble-components/build/generate-workers/index.ts

```diff
@@ -39,4 +42,10 @@ const renderFilePath: string = path.resolve(
     'matrix-renderer.ts'
 );
 
+const workersFilePath: string = path.resolve(
+    workersDirectory,
+    'types.ts'
+);
+
 writeFile(renderFilePath, fileContent);
+writeFile(workersFilePath, typesCode);
```

Comment:
> worker type export to the wafer. right now, I copy and paste the file contents

I think this approach seems okay enough for now, did you run into any issues / have any concerns with this approach?